### PR TITLE
fix(test): Adjust acceptance test related markup

### PR DESCRIPTION
**Pull request by victorvassilev**

### CODE REVIEW

File: src/Button/ButtonCustom/index.js [hunk 1]
```diff
@@ -24,15 +24,15 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function ButtonCustom({ label, onClick }) {
+function ButtonCustom({ label, onClick, id }) {
```
Comment (kduret) [inline src/Button/ButtonCustom/index.js]: I think it should not be a mandatory prop.
Use ...rest instead

Comment (kduret) [reply src/Button/ButtonCustom/index.js]:
```suggestion
function ButtonCustom({ label, onClick, ...rest }) {
```

File: src/Button/ButtonCustom/index.js [hunk 2]
```diff
@@ -24,15 +24,15 @@ const useStyles = makeStyles((theme) => ({
...
   const classes = useStyles();
-
   return (
     <Button
       variant="contained"
       color="secondary"
       className={classes.button}
       onClick={onClick}
+      id={id}
```
Comment (kduret) [inline src/Button/ButtonCustom/index.js]:
```suggestion
      {...rest}
```

**Comment by bdauria** [comment]
@victorvassilev any update on this?

**Pull request merged**